static-site/content/blog/2024-12-10-auspice-1password-issue.md

```diff
@@ -0,0 +1,10 @@
+---
+author: "The Nextstrain team"
+date: "2024-12-10"
+title: "Auspice slow/unresponsive with latest 1Password browser extension"
+sidebarTitle: "Externally-caused Auspice performance issues"
+---
+
+Scope: https://nextstrain.org and https://auspice.us, any other custom Auspice server
+
+We recently became aware that recent versions of the [1Password browser extension](https://1password.com/downloads/browser-extension) are interfering with Auspice visualizations, leading to slowness and in some cases unresponsiveness. We are actively working with the 1Password support team to resolve the problem. More information, workarounds, and updates can be found in a [GitHub issue](https://github.com/nextstrain/auspice/issues/1919).
```

Comment:
Absolutely fine as is, but maybe include a way to disable the extension here in the blog post, if there is a nice work-around? (I don't use the extension, but I should find a way to have it for Auspice development purposes)

Reply:
I decided not to add concrete steps in the blog post because it varies per browser. How about a note along the lines of:

```diff
… leading to slowness and in some cases unresponsiveness.
+ In the meantime, consider disabling the extension in your browser's settings
+ if you are experiencing difficulty viewing datasets.
More information, workarounds, and updates can be found in a [GitHub issue](https://github.com/nextstrain/auspice/issues/1919).
```